core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -556,21 +571,8 @@ private RecordCursorFactory createFullFatJoin(
                 for (int i = 0, n = listColumnFilterA.getColumnCount(); i < n; i++) {
                     int index = listColumnFilterA.getColumnIndexFactored(i);
                     final TableColumnMetadata m = ((AbstractRecordMetadata) slaveMetadata).getColumnMetadata(index);
-                    if (ColumnType.isSymbol(m.getType())) {
-                        metadata.add(
-                                slaveAlias,
-                                m.getName(),
-                                ColumnType.STRING,
-                                false,
-                                0,
-                                false,
-                                null
-                        );
-                        slaveTypes.add(ColumnType.STRING);
-                    } else {
-                        metadata.add(slaveAlias, m);
-                        slaveTypes.add(m.getType());
-                    }
+                    metadata.add(slaveAlias, m);
+                    slaveTypes.add(m.getType());
                     columnIndex.add(index);
                 }
             } else {
```

Comment:
I don't see this `else` to be test covered with Symbol columns.

Reply:
good one, let me have a look!

Reply:
hm, you are right. 
I ran all tests in the `io.questdb.test.griffin` package and the `else` branch was not executed *at all* 😱